pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -66,6 +67,7 @@ var _ = Describe("ImageUpload", func() {
 
 		dvCreateCalled  = &utils.AtomicBool{Lock: &sync.Mutex{}}
 		pvcCreateCalled = &utils.AtomicBool{Lock: &sync.Mutex{}}
+		dsCreateCalled  = &utils.AtomicBool{Lock: &sync.Mutex{}}
```

Comment:
/This construct looks weird.

Wouldn't something like this suffice to assert the expected verb was called?

```
Expect(kvtesting.FilterActions(&cdiClient.Fake, "create", "datasources")).To(HaveLen(1))
````